CHANGELOG.asciidoc

```diff
@@ -34,6 +34,7 @@ limitations under the License.
 * Improved behavior of `V()` and `E()` when `null` is an argument producing a filtering behavior rather than an exception.
 * Prevented metrics computation unless the traversal is in a locked state.
 * Added syntax to Gremlin grammar to explicitly define `byte`, `short` and `BigInteger`.
+* Added syntax to Gremlin grammar to allow construction of a reference `Vertex`.
```

Comment:
Shall we add an example in the changelog here to let the users know on what is possible in the syntax now?

Reply:
I think we will handle all the docs around gremlin-language with this: https://issues.apache.org/jira/browse/TINKERPOP-2644